package bespalhuk.logmee.service.exception

class UserAlreadyExistsException() : RuntimeException("User already exists.")
